lib/Elevate/Blockers/Repositories.pm

```diff
@@ -121,21 +121,18 @@ sub _yum_is_stable ($self) {
     if ( opendir( my $dfh, '/var/lib/yum' ) ) {
         my @transactions = grep { m/^transaction-all\./ } readdir $dfh;
         if (@transactions) {
-            ERROR('There are unfinished yum transactions remaining. Please address these before upgrading. The tool `yum-complete-transaction` may help you with this task.');
-            my $id = ref($self) . '::YumUnfinishedTransactions';
+            WARN('There are unfinished yum transactions remaining.');
 
-            $self->has_blocker(
-                'There are unfinished yum transactions remaining. Please address these before upgrading. The tool `yum-complete-transaction` may help you with this task.',
-                info => {
-                    name         => $id,
-                    error        => 'YUM has unfinished transactions',
-                    transactions => join( "\n", sort @transactions ),
-                },
-                blocker_id => $id,
-                quiet      => 1,
-            );
-
-            return 0;
+            if ( $self->is_check_mode() ) {
+                WARN('Unfinished yum transactions detected. Elevate will execute /usr/sbin/yum-complete-transaction --cleanup-only during upgrade');
+            }
+            else {
+                INFO('Cleaning up unfinished yum transactions.');
+                my $ret = $self->ssystem_capture_output(qw{/usr/sbin/yum-complete-transaction --cleanup-only});
+                if ( $ret->{status} != 0 ) {
+                    WARN( "Errors encountered running \"yum-complete-transaction\": " . $ret->{stderr} );
```

Comment:
We should die or throw a blocker since we are still in the blocker file if `yum-complete-transaction` fails.  This could be an indication of a much larger problem if it does fail.

Reply:
It could be that they just didn't have yum-utils installed.  I didn't think a problem with that utility itself should be a show-stopper.  Especially since leapp doesn't really care about unfinished yum transactions.

Reply:
Fair enough, I'll go ahead and approve this.